www/js/splash/remoteNotifyHandler.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * This module deals with handling specific push messages that open web pages
+ * or popups. It does not interface with the push plugin directly. Instead, it
+ * assumes that another module (currently `pushnotify`) deals with the plugin
+ * interface and emits a CLOUD_NOTIFICATION_EVENT when a push notification is
+ * received.
+ *
+ * This allows us to decouple the push handling logic from push notification
+ * interface. Note that the local notification is not currently decoupled since
+ * it only supports redirection to a specific app page. If the local
+ * notification handling gets more complex, we should consider decoupling it as well.
+ */
+import { EVENT_NAMES, subscribe } from '../customEventHandler';
+import { addStatEvent, statKeys } from '../plugin/clientStats';
+import { displayErrorMsg, logDebug } from '../plugin/logger';
+
+const options = 'location=yes,clearcache=no,toolbar=yes,hideurlbar=yes';
+
+/*
+TODO: Potentially unify with the survey URL loading
+*/
+/**
+ * @function launches a webpage
+ * @param url to open in the browser
+ */
+const launchWebpage = function (url) {
+  // THIS LINE FOR inAppBrowser
+  let iab = window['cordova'].InAppBrowser.open(url, '_blank', options);
+};
+
+/*
+TODO: replace popup with something with better UI
+*/
+
+/**
+ * @function launches popup
+ * @param title string text for popup title
+ * @param text string text for popup bode
+ */
+const launchPopup = function (title, text) {
+  // THIS LINE FOR inAppBrowser
+  displayErrorMsg(text, title);
```

Comment:
I know we don't have a good popup mechanism right now so you used this temporarily, but this would actually log an error to Logger plugin and I don't think that's appropriate here

Either use `window.alert` directly or add a function to `logger.ts` like `displayPopup`, which could be called by `displayErrorMsg`

And I think you can do that call directly from line 74  -- this 1-line function serves no purpose

Reply:
done in [c25ebb6](https://github.com/e-mission/e-mission-phone/pull/1093/commits/c25ebb6f71f20d24f14b135186da66013b6806f8)